src/client/Microsoft.Identity.Client/Platforms/iOS/WebviewBase.cs

```diff
@@ -37,15 +40,28 @@ public WebviewBase()
             RequestContext requestContext,
             CancellationToken cancellationToken);
 
-        public static bool ContinueAuthentication(string url)
+        public static bool ContinueAuthentication(string url, Core.ICoreLogger unreliableLogger)
```

Comment:
What exactly is unreliable referring to in this context?

Reply:
Our logger is tied to a request - we create a new logger for each request. This way we always log the correlation ID. 
But since these authentication callbacks are static, we cannot inject the logger into them via the constructor. Instead, I made it so that each request sets the logger static property. 

This is potentially unreliable if multiple requests are fired at the same time - the logger will simply be the one from the last fire request. So we cannot necessarily trust the correlation IDs. That's why I was thinking `unreliable`. But maybe a better name can be found ... suggestions? (naming is hard)

Reply:
I think latestRequestLogger or lastRequestLogger might be better at explaining that this is only logging the last request. otherwise add a comment to explain so that people are aware of this during debugging